pkg/scheduler/scheduler_test.go

```diff
@@ -828,7 +881,7 @@ func TestSchedulerWithVolumeBinding(t *testing.T) {
 				FindBoundSatsified:   false,
 			},
 			eventReason: "FailedScheduling",
-			expectError: makePredicateError("1 node(s) didn't find available persistent volumes to bind, 1 node(s) had volume node affinity conflict"),
+			expectError: makePredicateError("1 node(s) had volume node affinity conflict"),
```

Comment:
`didn't find available persistent volumes to bind` is removed because only the first reason is used by the new framework status